index.js

```diff
@@ -214,6 +214,10 @@ Pica.prototype.resize = function (from, to, options) {
   opts.width    = from.naturalWidth || from.width;
   opts.height   = from.naturalHeight || from.height;
 
+  if (to.width === 0 || to.height === 0) {
+    return Promise.reject(`Invalid output size: ${to.width}x${to.height}`);
```

Comment:
- Missed `new Error`
- Add comment why this check required

Reply:
Done :)